src/wp-admin/includes/class-wp-site-health-auto-updates.php

```diff
@@ -74,9 +74,10 @@ public function test_constants( $constant, $value ) {
 		if ( defined( $constant ) && ! in_array( constant( $constant ), $acceptable_values, true ) ) {
 			return array(
 				'description' => sprintf(
-					/* translators: %s: Name of the constant used. */
-					__( 'The %s constant is defined and enabled.' ),
-					"<code>$constant</code>"
+					/* translators: 1: Name of the constant used. 2: Value of the constant used. */
+					__( 'The %1$s constant is defined as $2$s' ),
```

Comment:
The second translator argument here is `$2$s`, and should be `%2$s`.

```suggestion
					__( 'The %1$s constant is defined as %2$s' ),
```